tools/slicec-cs/src/main.rs

```diff
@@ -57,7 +58,15 @@ pub fn main() {
         }
     }
 
-    std::process::exit(i32::from(compilation_state.emit_diagnostics(slice_options)))
+    // Emit diagnostics and totals.
+    let exit_code = i32::from(compilation_state.emit_diagnostics(slice_options));
+
+    // If the metrics flag is set mode is enabled, output additional compilation information.
```

Comment:
The comment needs to be fixed.

It's unclear why we include the exit code in the output. Presumably the caller always gets the exit code and is free to do whatever it wants with it.